erpnext/selling/doctype/sales_order/sales_order.json

```diff
@@ -282,6 +282,8 @@
    "width": "100px"
   },
   {
+   "fetch_from": "customer.tax_id",
+   "fetch_if_empty": 1,
```

Comment:
fetch_if_empty should no be 1. Since this is a read_only field it should always be fetched from Customer master

Reply:
@deepeshgarg007 Done!